<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CodeDialog</class>
 <widget class="QDialog" name="CodeDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>13</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Edit Codes</string>
  </property>
  <widget class="Q3ListBox" name="codeListBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>250</width>
     <height>211</height>
    </rect>
   </property>
  </widget>
  <widget class="Q3GroupBox" name="groupBox2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>551</width>
     <height>180</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="textLabel1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Code</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="textLabel2_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>220</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Description</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditCode">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>60</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="colorGroupLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Color Code</string>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="codeDescrip">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>50</y>
      <width>310</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="codeSaveButt">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Sa&amp;ve</string>
    </property>
    <property name="shortcut">
     <string>Alt+V</string>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="codeDelButt">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>140</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&amp;Delete</string>
    </property>
    <property name="shortcut">
     <string>Alt+D</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QComboBox" name="colorComboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>120</width>
      <height>30</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>0-69</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>70-79</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>80-89</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>90-99</string>
     </property>
    </item>
   </widget>
   <widget class="QCheckBox" name="codePair">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>140</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Pair?</string>
    </property>
   </widget>
   <widget class="Q3ButtonGroup" name="codeEditModeGroup">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>131</width>
      <height>120</height>
     </rect>
    </property>
    <property name="title">
     <string>Edit Mode</string>
    </property>
    <widget class="QRadioButton" name="editColorButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Color</string>
     </property>
     <property name="shortcut">
      <string>Alt+C</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QRadioButton" name="editSexButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Sex</string>
     </property>
     <property name="shortcut">
      <string>Alt+S</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="editFlawButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>&amp;Flaw</string>
     </property>
     <property name="shortcut">
      <string>Alt+F</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="printListButton">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>440</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>&amp;Print</string>
   </property>
   <property name="shortcut">
    <string>Alt+P</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3Frame</class>
   <extends>QFrame</extends>
   <header>Qt3Support/Q3Frame</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ButtonGroup</class>
   <extends>Q3GroupBox</extends>
   <header>Qt3Support/Q3ButtonGroup</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Q3ListBox</class>
   <extends>Q3Frame</extends>
   <header>q3listbox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>lineEditCode</tabstop>
  <tabstop>codeDescrip</tabstop>
  <tabstop>codePair</tabstop>
  <tabstop>editColorButton</tabstop>
  <tabstop>codeSaveButt</tabstop>
  <tabstop>codeDelButt</tabstop>
  <tabstop>codeListBox</tabstop>
 </tabstops>
 <includes>
  <include location="global">vector</include>
  <include location="local">types/animalcode.h</include>
  <include location="local">dao/animalcodedao.h</include>
  <include location="local">dao/commissiondao.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
